src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext, useEffect } from 'react';
+import cn from 'classnames';
+import { appContext } from '../Context/Context';
+
+export const Errors: React.FC = () => {
+  const { errors, setErrors } = useContext(appContext);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setErrors(null);
+    }, 3000);
+  }, [errors]);
```

Comment:
React Hook useEffect has a missing dependency: 'setErrors'. Either include it or remove the dependency array